src/LiveSplit.PossibleTimeSave/UI/Components/PossibleTimeSaveSettings.Designer.cs

```diff
@@ -451,23 +429,34 @@ private void InitializeComponent()
             // 
             this.chkTotalTimeSave.Anchor = ((System.Windows.Forms.AnchorStyles)((System.Windows.Forms.AnchorStyles.Left | System.Windows.Forms.AnchorStyles.Right)));
             this.chkTotalTimeSave.AutoSize = true;
-            this.chkTotalTimeSave.Location = new System.Drawing.Point(336, 100);
-            this.chkTotalTimeSave.Margin = new System.Windows.Forms.Padding(10, 5, 4, 5);
+            this.chkTotalTimeSave.Location = new System.Drawing.Point(224, 64);
+            this.chkTotalTimeSave.Margin = new System.Windows.Forms.Padding(7, 3, 3, 3);
             this.chkTotalTimeSave.Name = "chkTotalTimeSave";
-            this.chkTotalTimeSave.Size = new System.Drawing.Size(354, 24);
+            this.chkTotalTimeSave.Size = new System.Drawing.Size(235, 17);
             this.chkTotalTimeSave.TabIndex = 5;
             this.chkTotalTimeSave.Text = "Show Total Possible Time Save";
             this.chkTotalTimeSave.UseVisualStyleBackColor = true;
             // 
+            // rdoMilliseconds
+            // 
+            this.rdoMilliseconds.Anchor = ((System.Windows.Forms.AnchorStyles)((System.Windows.Forms.AnchorStyles.Left | System.Windows.Forms.AnchorStyles.Right)));
+            this.rdoMilliseconds.AutoSize = true;
+            this.rdoMilliseconds.Location = new System.Drawing.Point(330, 31);
+            this.rdoMilliseconds.Name = "rdoMilliseconds";
+            this.rdoMilliseconds.Size = new System.Drawing.Size(105, 17);
+            this.rdoMilliseconds.TabIndex = 3;
+            this.rdoMilliseconds.TabStop = true;
+            this.rdoMilliseconds.Text = "Milliseconds";
+            this.rdoMilliseconds.UseVisualStyleBackColor = true;
+            // 
             // PossibleTimeSaveSettings
             // 
-            this.AutoScaleDimensions = new System.Drawing.SizeF(9F, 20F);
+            this.AutoScaleDimensions = new System.Drawing.SizeF(6F, 13F);
             this.AutoScaleMode = System.Windows.Forms.AutoScaleMode.Font;
             this.Controls.Add(this.tableLayoutPanel1);
-            this.Margin = new System.Windows.Forms.Padding(4, 5, 4, 5);
             this.Name = "PossibleTimeSaveSettings";
-            this.Padding = new System.Windows.Forms.Padding(10, 11, 10, 11);
-            this.Size = new System.Drawing.Size(714, 569);
+            this.Padding = new System.Windows.Forms.Padding(7, 7, 7, 7);
+            this.Size = new System.Drawing.Size(476, 370);
```

Comment:
The size of the entire form changing is an indication that the scale was not set to 100%

Reply:
I was using 100% scaling the whole time though? Idk I'll fix it anyways

Reply:
It seems like the last person who changed this file may not have been using 100% scaling. This PR is fine then